vision_evaluation/evaluators.py

```diff
@@ -417,13 +483,21 @@ def _calculate_average_precision(is_correct, probabilities, true_num, average='m
         return sm.average_precision_score(is_correct, probabilities, average=average) * recall
 
     def get_report(self, **kwargs):
+        """
+        Args:
+            labels : If report_per_class, pass the list of class names the same order as the ground_truths label index
+        """
         average = kwargs['average'] if 'average' in kwargs else 'macro'
```

Comment:
not your change, but why don't we add 'average' into the method argument?

Reply:
removed